routes.js

```diff
@@ -1,86 +1,86 @@
-// const express = require('express');
-// const createMyCustomApi = express.Router();
-// const fs = require('fs');
+const express = require('express');
+const createMyCustomApi = express.Router();
+const fs = require('fs');
 
 
 
 
-// // Define routes
-// createMyCustomApi.get('/getAllMembers', (req, res) => {
-//   fs.readFile("./members_Array.json",  (err, data) => {
-//     if(err){
-//       res.status(500).send(err.message)
-//     }
-//     res.status(200).send(data.toString());
-//   });
-// });
+// Define routes
+createMyCustomApi.get('/getAllMembers', (req, res) => {
+  fs.readFile("./members_Array.json",  (err, data) => {
+    if(err){
+      res.status(500).send(err.message)
+    }
+    res.status(200).send(data.toString());
+  });
+});
 
 
 
-// createMyCustomApi.post('/addNewMember', (req, res) => {
-//   let data = fs.readFileSync("./members_Array.json"); //diavazoume to array pou exoume hdh
-//   let myObject = JSON.parse(data); // to knaoume js object me parsing
-//   let newData = req.body; // dilwnoume newData to reqbody apo postman
-//     myObject.push(newData); // pusharo sto yparoxn js object to body
-//       console.log(myObject);
-//     let newArray = JSON.stringify(myObject);
-//     fs.writeFile("./members_Array.json", newArray, err => {
-//       if(err) throw err;
-//       console.log(newArray)
-//     });
-//     res.sendStatus(200);
-// });
+createMyCustomApi.post('/addNewMember', (req, res) => {
+  let data = fs.readFileSync("./members_Array.json"); //diavazoume to array pou exoume hdh
+  let myObject = JSON.parse(data); // to knaoume js object me parsing
+  let newData = req.body; // dilwnoume newData to reqbody apo postman
+    myObject.push(newData); // pusharo sto yparoxn js object to body
+      console.log(myObject);
+    let newArray = JSON.stringify(myObject);
+    fs.writeFile("./members_Array.json", newArray, err => {
+      if(err) throw err;
+      console.log(newArray)
+    });
+    res.sendStatus(200);
+});
 
 
-// createMyCustomApi.put('/updateExistMember', (req, res) => {
-//   const {memberId, role , firstname, lastname} = req.body
-//   if(!memberId) {
-//     return res.status(400).send('Member ID is required');
-//   }
-//   const memberIndex = data.findIndex(member => member.memberId === memberId);
-//   if (memberId === -1) {
-//     return res.status(404).send("Member not found");
-//   }
-//   const updatedMember = {
-//     memberId,
-//     role : role,
-//     firstname : firstname ,
-//     lastname : lastname
-//   };
+createMyCustomApi.put('/updateExistMember', (req, res) => {
+  const {memberId, role , firstname, lastname} = req.body
+  if(!memberId) {
+    return res.status(400).send('Member ID is required');
+  }
```

Comment:
Great job adding validation!
Thinking if you need a validation for the `post` endpoint too?

Reply:
Done